### DISCUSSION TITLE
Importing CAM User and Groups using TM1py

### DISCUSSION THREAD

**Discussion by Mr-SabyasachiBose** [cubewise-code/tm1py#1193] [General]
**Describe what did you try to do with TM1py**

Is it possible to import users and groups from CAM using TM1py, like we do in Architect? Cognos also supports REST API, hence wondering if TM1py can connect to both for user/group updates. Importing from and Exporting to CAM, if both possible it would be nice. Currently we use some JAR files with API calls to create/delete groups and user in groups in CAM based on TM1 cube values.

**Comment by rclapp** [comment]
It doesn't as if today. We love contributions though.

On Oct 18, 2024 1:25 AM, Sabyasachi Bose ***@***.***> wrote:

Describe what did you try to do with TM1py

Is it possible to import users and groups from CAM using TM1py, like we do in Architect? Cognos also supports REST API, hence wondering if TM1py can connect to both for user/group updates. Importing from and Exporting to CAM, if both possible it would be nice. Currently we use some JAR files with API calls to create/delete groups and user in groups in CAM based on TM1 cube values.
